Kernel/Arch/aarch64/Exceptions.cpp

```diff
@@ -84,30 +85,53 @@ static void setup_el1()
     Aarch64::Asm::load_el1_vector_table(&vector_table_el1);
 }
 
-void initialize_exceptions(u32 cpu)
+void initialize_exceptions()
 {
     auto base_exception_level = Aarch64::Asm::get_current_exception_level();
 
     if (base_exception_level > Aarch64::Asm::ExceptionLevel::EL3) {
-        PANIC("CPU[{}]: Started in unknown EL{}", cpu, static_cast<u8>(base_exception_level));
+        panic_without_mmu("Started in unknown EL (Greater than EL3)"sv);
     } else if (base_exception_level < Aarch64::Asm::ExceptionLevel::EL1) {
-        PANIC("CPU[{}]: Started in unsupported EL{}", cpu, static_cast<u8>(base_exception_level));
+        panic_without_mmu("Started in unsupported EL (Less than EL1)"sv);
     } else {
-        dbgln("CPU[{}]: Started in EL{}", cpu, static_cast<u8>(base_exception_level));
+        StringView message;
```

Comment:
very nit: could get rid of this variable and just call `dbgln_without_mmu` in each branch directly. But fine as-is too.

Reply:
Done :)